tasks/add-bg-music.md

```diff
@@ -0,0 +1,31 @@
+
+
+---
+
+# Labyrinth: Add background music
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+To make the game more interesting, please add background music.
+
+Requirements:
+- Add background music eg:Recording of your favorite song, it is longer. sing/ guitar/ piano/composition.
+- Make sure it play when user comes onto the page and starts playing
+- Add the music to the HTML file. Note this can be done through HTML however you would have to add a music file to the repository
```

Comment:
here, we will edit it again to point to the API.

Reply:
I got all your comments except this one, can you tell what I should write